src/DevBetterWeb.Web/Controllers/VideosController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using DevBetterWeb.Vimeo.Services.VideoServices;
+using DevBetterWeb.Web.Models.Vimeo;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace DevBetterWeb.Web.Controllers
+{
+  [Route("videos")]
+  [ApiController]
+  public class VideosController : Controller
+  {
+    private readonly IMapper _mapper;
+    private readonly GetAllVideosService _getAllVideosService;
+
+    public VideosController(IMapper mapper, GetAllVideosService getAllVideosService)
+    {
+      _mapper = mapper;
+      _getAllVideosService = getAllVideosService;
+    }
+
+    [HttpPost("list")]
+    public async Task<IActionResult> ListAsync()
+    {
+      var draw = Request.Form["draw"].FirstOrDefault();      
+      var length = Request.Form["length"].FirstOrDefault();
```

Comment:
Why not use model binding for these values instead of reading from `Request.Form`?

Reply:
added.